checker/src/main/java/org/checkerframework/checker/nullness/NullnessChecker.java

```diff
@@ -31,6 +32,7 @@
   NullnessChecker.LINT_TRUSTARRAYLENZERO,
   NullnessChecker.LINT_PERMITCLEARPROPERTY
 })
+@StubFiles({"junit-assertions.astub"})
```

Comment:
This causes the stub file to be parsed every time the checker is used.  We should probably put it somewhere to be passed only if using JUnit.

Reply:
@smillst I'm concerned that it will not be very discoverable by users if they have to supply a special command line option explicitly.
Would this be acceptable if the stub file were pared down to be minimal?
Or should we document `@StubFiles` as to be used only for JDK annotations and not for any other libraries?